packages/amplify_core/lib/src/config/analytics/pinpoint_config.dart

```diff
@@ -35,6 +36,7 @@ class PinpointPluginConfig
   const PinpointPluginConfig({
     required this.pinpointAnalytics,
     required this.pinpointTargeting,
+    this.autoFlushEventsInterval = const Duration(seconds: 30),
```

Comment:
What is the current default in 0.6.x?

Reply:
For 0.6.x, we rely on Android and iOS.  On Android it is 30 seconds, on iOS it is 60 seconds.  

I asked the teams why it was 30 or 60 seconds and no one knew... 

I spoke with Abdallah this morning about this and we confirmed together that 30 seconds should be okay.  I can share the reasoning here if you'd like.